mobile/src/lib/cozy-helper.js

```diff
@@ -34,3 +34,16 @@ export const initBar = () => {
     lang: 'en'
   })
 }
+
+export const isClientRegistered = async (client) => {
+  return await cozy.client.auth.getClient(client).then(client => true).catch(() => false)
```

Comment:
I added some code in cozy-client-js (see below https://github.com/cozy/cozy-client-js/issues/105) to handle this edge case.